rpc/nns/hashes.go

```diff
@@ -27,3 +35,28 @@ func InferHash(sg ContractStateGetter) (util.Uint160, error) {
 
 	return c.Hash, nil
 }
+
+// ResolveFSContract is a convenience method that doesn't exist in the NNS
+// contract itself (it doesn't care which data is stored there). It assumes
+// that contracts follow the [ContractTLD] convention, gets simple contract
+// names (like "container" or "netmap") and extracts the hash for the
+// respective NNS record in any of the formats (of which historically there's
+// been a few).
+func (c *ContractReader) ResolveFSContract(name string) (util.Uint160, error) {
+	strs, err := c.Resolve(name+"."+ContractTLD, TXT)
```

Comment:
IIRC, NNS's "resolve" method is able to return `stackitem.Null`, and `c.Resolve` isn't able to properly handle this. Don't we need to add some comment about it? Although the `ResolveFSContract` return value is either hash or error, and this behaviour should be OK. But maybe we also need to check some other usages of `c.Resolce`.

Reply:
> eturn value is either hash or error

That's exactly why it's the the way it is now.